### PR TITLE
Tested for lost password

### PR DESCRIPTION
**Pull request by tazeek**
Test case should cover:

- Navigating to 'Lost Password' page
- Receiving email along with token
- Validating the new password
- Passwords must contain lowercase, uppercase and number(s)
- Passwords must match
- If user has a real money account, get date of birth
- Going back to 'send token' page, assuming the original token has expired or is invalid

### CODE REVIEW

**Review by bijan2007 — CHANGES_REQUESTED** [review]
Please commit only the files related to your changes

**Review by ridhomard — CHANGES_REQUESTED** [review]
Need to add test cases .xml

**Comment by tazeek** [comment]
Performing changes and adding test cases now

**Review by ridhomard — CHANGES_REQUESTED** [review]
1. Should give more form validation 
2. Should minimise use of manual input (should use constant)
3. Retry link need more validation 
4. User assert.

**Review by bijan2007 — CHANGES_REQUESTED** [review]
Please resolve the conflicts and travis test should be passing

**Review by bijan2007 — APPROVED** [review]

File: src/test/java/testCases/Validate_LostPassword.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	public void setUp() {
+    	ChromeDriverManager.getInstance().setup();
+    	driver = new ChromeDriver(); 
+    	driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+    	driver.get(Constant.URL_Binary);
+    }
+	
+	//Test to navigate to lost password page
+	@Test(priority=0)
+	public void navigateToLostPasswordPage() {
```
Comment (bijan2007) [inline src/test/java/testCases/Validate_LostPassword.java]: Please use navigation methods

File: src/test/java/utility/Constant.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 public class Constant {
 	//Create constant variables
 	public static final String URL = "https://staging.binary.com";
+	public static final String URL_Binary = "https://binary.com";
```
Comment (bijan2007) [inline src/test/java/utility/Constant.java]: We should check against staging and not production

**Review by bijan2007 — CHANGES_REQUESTED** [review]
Please look at the comment to test against staging

**Comment by tazeek** [comment]
Changed url to staging instead of production

**Review by bijan2007 — APPROVED** [review]

**Comment by bijan2007** [comment]
Closing this PR as it is outdated

**Pull request closed without merging**